OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,201 @@
+
+Please refer README.md under docker/OracleDatabase for details on how to build Oracle Database image.
+https://github.com/oracle/docker-images/tree/master/OracleDatabase
+
+
+## Running Oracle Database in a Docker container
+
+Create an environment file db.env.list
+
+    ORACLE_SID=<DB SID>
+    ORACLE_PDB=<PDB ID>
+    ORACLE_PWD=<password>
+    
+Sample Data will look like this:
+
+    ORACLE_SID=soadb
+    ORACLE_PDB=soapdb
+    ORACLE_PWD=Welcome1
+
+Sample Command to Start the Database is as follows
+
+     $ docker run --name soadb  --network=SOANet -p 1521:1521 -p 5500:5500 -v /scratch/DockerVolume/SOAVolume/DB:/opt/oracle/oradata --env-file ./db.env.list  oracle/database:12.2.0.1-ee
+     
+The above command starts a DB container attaching to a network and mounting a host directory as `/opt/oracle/oradata` for persistence. 
+It maps the containers 1521 and 5500 port to respective host port such that the services can be accessible outside of localhost.
+
+
+## SOA Distributable version 12.2.1.3 Docker image Creation and Running
+
+To build a SOA image either you can start from building Oracle JDK and Oracle FMW Infrastrucure image or use the already available Oracle FMW Infrastructure image. The FMW Infrastructure image is available in the Oracle Container Registry and can be pulled from there. If you plan to use the Oracle FMW Infrastructure image from the Oracle Container Registry, you can skip the next two steps and continue with "Building a Docker Image for SOA".
+
+
+## Building Oracle JDK (Server JRE) base image
+
+Please refer README.md under docker/OracleJava for details on how to build Oracle Database image.
+
+https://github.com/oracle/docker-images/tree/master/OracleJava
+
+
+## Building Oracle FMW Infrastructure Docker Install Image
+
+Please refer README.md under docker/OracleFMWInfrastructure for details on how to build Oracle FMW Infrastructure image.
+
+https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure
+
+
+
+## Building Docker Image for SOA
+
+IMPORTANT: To build the SOA image, you must first download the Oracle SOA Suite 12.2.1.3 binary and drop in folder `../OracleSOASuite/dockerfiles/12.2.1.3`. The binaries can be downloaded from the Oracle Software Delivery Cloud (https://edelivery.oracle.com). Search for "Oracle SOA Suite" and "Oracle Service Bus", download fmw_12.2.1.3.0_soa.jar and fmw_12.2.1.3.0_osb.jar and drop it under `dockerfiles/12.2.1.3`. 
+
+* [Oracle Software Delivery Cloud](https://edelivery.oracle.com)
+
+$ sh buildDockerImage.sh -v 12.2.1.3
+
+   Usage: buildDockerImage.sh -v [version]
+   Builds a Docker Image for Oracle SOA Suite.
+
+
+Verify you now have the image `oracle/soa:12.2.1.3` in place with 
+
+$ docker images
```

Comment:
I would include `| grep "soa"` to this command because a user may have lots and lots of images.